app/client/models/DocPageModel.ts

```diff
@@ -294,7 +294,7 @@ Recovery mode opens the document to be fully accessible to owners, and inaccessi
 It also disables formulas. [{{error}}]", {error: err.message})
             : isDenied
               ? t('Sorry, access to this document has been denied. [{{error}}]', {error: err.message})
-              : t("Document owners can attempt to recover the document. [{{error}}]", {error: err.message})
+              : t("Please reload the document and if the error persist, contact the document owners to attempt a document recovery. [{{error}}]", {error: err.message})
```

Comment:
"if the error persist" -> "if the error persists"

Also the line has become too long, a linter is complaining.

Reply:
Oops, thanks!